.eslintrc.js

```diff
@@ -9,6 +9,7 @@ module.exports = {
   },
   extends: "eslint:recommended",
   env: {
+    es6: true,
```

Comment:
I used async validation of `ember-changeset-validation`, which returns promise. ES lint throws `undef` error unless es6 env is set as true.